src/components/status/status.ts

```diff
@@ -0,0 +1,76 @@
+import { CSSResultGroup, html, LitElement, nothing, TemplateResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+import { setAttribute } from '../core/dom';
+import type { TitleLevel } from '../title';
+
+import style from './status.scss?lit&inline';
+
+import '../icon';
+import '../title';
+
+const statusTypes = new Map([
+  ['info', 'circle-information-small'],
+  ['success', 'circle-tick-small'],
+  ['warning', 'circle-exclamation-point-small'],
+  ['error', 'circle-cross-small'],
+]);
+
+/**
+ * Displays a message to the user's attention.
+ *
+ * @slot - Use the unnamed slot to add content to the status message.
+ * @slot title - Use this to provide a title for the status (optional).
+ */
+@customElement('sbb-status')
+export class SbbStatus extends LitElement {
+  public static override styles: CSSResultGroup = style;
+
+  /**
+   * The type of the status.
+   */
+  @property({ reflect: true }) public type?: 'info' | 'success' | 'warning' | 'error' = 'info';
+
+  /**
+   * The version of the status.
+   */
+  @property({ reflect: true, attribute: 'extended', type: Boolean }) public hasTitle: boolean;
```

Comment:
The different attribute name is a specific request?

Reply:
I took hasTitle from Figma, but I think we can use the name we prefer, or ask Lukas (or create a poll 😃 )

Reply:
I would prefer to have it aligned, so maybe like this?

```suggestion
  @property({ reflect: true, attribute: 'extended', type: Boolean }) public extended: boolean;
```

Reply:
In order to have them more self-explanatory, I now replaced them with `hasTitle` / `'has-title'`